lobby/src/main/java/org/triplea/lobby/server/controller/rest/ControllerHub.java

```diff
@@ -0,0 +1,17 @@
+package org.triplea.lobby.server.controller.rest;
+
+import static spark.Spark.path;
+
+import org.triplea.lobby.server.config.LobbyConfiguration;
+import org.triplea.lobby.server.db.Database;
+
+/**
+ * Main Routing class to connect all Controllers together.
+ */
+public class ControllerHub {
+
+  public static void initializeControllers(final LobbyConfiguration configuration) {
+    final Database database = new Database(configuration);
+    path("/api/v0", new ModeratorActionController(database)::initializeRoutes);
```

Comment:
We have http-lobby which is a spark server, it was set to replace the lobby entirely. I'm not sure it makes sense to have http-server if we're going to have lobby also be a spark server too.

Reply:
I wasn't aware that the `http-server` module aims to replace the lobby entirely, mainly because currently the only classes that are implemented relate to the error reporting mechanism, which I'm not sure if it should be part of the same server as the lobby